packages/grid/_modules_/grid/components/panel/ColumnsPanel.tsx

```diff
@@ -78,8 +78,12 @@ export function ColumnsPanel() {
     [columns, searchValue],
   );
 
+  const timeoutFocus = React.useRef<any>(0);
```

Comment:
Do we need to store 0?

```suggestion
  const timeoutFocus = React.useRef<any>();
```